metadata.yaml

```diff
@@ -4,6 +4,8 @@ description: |
   reducing the need for boilerplate code.
 
   https://github.com/metacontroller/metacontroller
+series: [kubernetes]
+min-juju-version: "2.9.0"
```

Comment:
`series: [kubernetes]` will make Juju run this as podspec, which will then break the charm because it depends on using `--trust` which only works with sidecar. 

Instead, I believe you need
```
assumes:
    - juju >= 2.9
```

Reply:
To clarify the above comment as it was a bit confusing, this charm relies on creating objects in k8s.  By default, deployed charms do now have the RBAC to support that and thus require elevated privileges.  These privileges come from deploying with `--trust`, which is a feature that came in with sidecar charms (eg, `--trust` **does not** elevate the privileges of a charm).  

On the other side, including `series: [kubernetes]` will hint to juju that this charm is a **podspec** charm, so including that in the metadata will break the `--trust` feature.  In the case here it is a really subtle break, as we don't actually need a sidecar and it might be harder to notice.

Reply:
Hi @ca-scribner, just made the change, thanks for the explanation.